civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -377,4 +463,154 @@ $carousel-box-shadow-color: rgba(#3f3c55,0.3);
+    .views-slideshow-controls-text-previous,
+    .views-slideshow-controls-text-next {
+      display: none;
+    }
+
+    .views-slideshow-cycle-main-frame {
+      margin: 0;
+      height: 400px;
+
+      .views-slideshow-cycle-main-frame-row {
+        .views-slideshow-cycle-main-frame-row-item {
+          margin: 0;
+
+          .views-field-field-slideshow-image {
+            width: 100% !important;
+
+            img {
+              height: 200px !important;
+              width: 100% !important;
+            }
+          }
+
+          .views-field-title {
+            left: 0;
+            top: 240px;
+            width: 100%;
+          }
+
+          .views-field-field-description {
+            left: 0;
+            top: 290px;
+            width: 100%;
+          }
+        }
+      }
+    }
+  }
+}
+
+.page-customize-onboarding-wizard {
+  #onboarding-customization-page {
+    position: relative;
+
+    h1 {
+      margin-top: 50px;
+    }
+
+    .region {
+      background: $light;
+    }
+
+    .drop-layer {
+      background: $light;
+      height: 140px;
+      top: 25px;
+
+      i {
+        color: inherit;
+      }
+    }
+
+    #edit-civihr-onboarding-organization-logo-fid-ajax-wrapper {
+      input.form-file {
+        height: 140px;
+        margin-bottom: 15px;
+        min-width: 120px;
+        width: 100%;
+      }
+
+      .help-block {
+        display: none;
+      }
+    }
+
+    #edit-civihr-onboarding-organization-logo-fid-ajax-wrapper,
+    .form-item-civihr-onboarding-carousel-options {
+      height: 250px;
+      margin-left: 100px;
+      padding-top: 50px;
+      width: 350px;
+    }
+
+    .form-actions {
+      padding-bottom: 20px;
+      padding-right: 20px;
+    }
+
+    .form-managed-file {
+      input[type='submit'],
+      .ajax-progress {
+        float: right;
+      }
+    }
+
+    .form-item-civihr-onboarding-welcome-text,
+    .form-item-civihr-onboarding-carousel-options,
+    .form-item-civihr-onboarding-intro-text {
+      margin-bottom: 100px;
+    }
+
+    .form-item-civihr-onboarding-carousel-options {
+      .form-type-checkbox {
+        margin-bottom: 10px !important;
+      }
+    }
+
+    .form-item-civihr-onboarding-welcome-text,
+    .form-item-civihr-onboarding-intro-text {
+      height: 200px;
+      margin-left: 100px;
+      padding-top: 50px;
+      width: 350px;
+
+      .help-block {
+        display: none;
+      }
+
+      .form-textarea-wrapper {
+        height: 100%;
+
+        textarea {
+          height: 100%;
+          resize: none;
+        }
+      }
+    }
+
+    .onboarding-customize-logo {
+      width: 440px;
+      position: absolute;
+      top: 290px;
+      right: 100px;
+    }
+
+    .onboarding-customize-features {
+      width: 440px;
+      position: absolute;
+      top: 850px;
+      right: 100px;
+    }
+
+    .onboarding-customize-welcome {
+      width: 440px;
+      position: absolute;
+      top: 1200px;
```

Comment:
Is there a reason why these elements are being positioned absolute instead of following a natural flow (ie, only using margin and padding)?

What would happen if the customize features content expands? would it overlap with the customize welcome section?

Reply:
Unfortunately this was the only way I found to make this work, because the images are injected in the page using JS, and they do not belong to the original flow of the HTML. 